### DISCUSSION TITLE
I have been billed twice

### DISCUSSION THREAD

**Discussion by AlexanderCGO2** [AntlerVC/firetable#1025] [Support / Q&A] [answered]
Hi there, you've billed me 2 times the subscription for 1 month....please help!

**Comment by bofeiw** [comment, marked as answer]
Hi @AlexanderCGO2, I can see you were charged $24. 

Our [pricing model](https://www.rowy.io/pricing) is $12 per seat per month. Your workspace have 2 members so you are charged 2 * $12 = $24.

Hope this answers your question.
